modules/integration/tests-common/integration-tests-utils/src/main/resources/mapper.properties

```diff
@@ -138,3 +138,14 @@ mb.dlc.browse.table.reroute.confirm=/html/body/div[3]/div[2]/button
 mb.dlc.rerouted.message.id=//*[@id="workArea"]/table[2]/tbody/tr[1]/td[2]
 mb.dlc.rerouted.queue.table=//*[@id="workArea"]/table[2]/tbody
 
+#Topics
+
+mb.add.topics.page.header.xpath=//*[@id="workArea"]/h2
+mb.add.topics.page.topic.name.field.id=topic
+mb.add.topics.page.add.button.xpath=//*[@id="permissionTable"]/tbody/tr[3]/td/input
+home.mb.topics.add.xpath=//*[@id="menu"]/ul/li[5]/ul/li[4]/ul/li[2]/a
+home.mb.topics.browse.xpath=//*[@id="menu"]/ul/li[5]/ul/li[4]/ul/li[1]/a
+mb.topic.browse.page.header.xpath=//*[@id="workArea"]/h2
+mb.topic.browse.topictree=//*[@id="topicTree"]
+home.mb.topics.add.without.queue.xpath=//*[@id="menu"]/ul/li[5]/ul/li[2]/ul/li[2]/a
+home.mb.topics.browse.without.queue.xpath=//*[@id="menu"]/ul/li[5]/ul/li[2]/ul/li[1]/a
```

Comment:
We can generalize these expressions to use unique element IDs (e.g table with name "content") . This could minimize rework if the UI is changed someday. Food for thought :)